src/ducks/event.js

```diff
@@ -22,13 +23,15 @@ export const setOrderBy = createAction('SET_EVENT_ORDER_BY');
 
 export default (state = defaultState(), action) => {
   const { type, payload } = action;
+
   switch (type) {
     case 'GET_EVENTS_FULFILLED':
       return state
-        .set('count', payload.count)
-        .set('next', payload.next)
-        .set('previous', payload.previous)
-        .update('events', events => events.merge(payload.results));
+        .set('count', payload.data.count)
+        .set('next', payload.data.next)
+        .set('previous', payload.data.previous)
+        .set('ordering', ordering())
```

Comment:
Because if sorted in any way and we fetch more events, it causes them to be added to the sorted listed wherever it is sorted by and confusing for the user because he/she will not see the new events without specifically looking for them.